.github/workflows/publish.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         python: [3.6, 3.7, 3.8, 3.9]
-        django: [2.2.*, 3.0.*, 3.1.*]
+        django: [2.2.*, 3.1.*, 3.2.*]
```

Comment:
4.0a1?

Reply:
Should probably replace this with the tox version like the workflow which runs on PRs. 

🕵️

Will investigate.

Reply:
Ok, I've updated this. I guess there's only one way to test this! 😬